### PR TITLE
Housing Lottery Add Hall Card

### PR DESCRIPTION
**Pull request by ArabellaJi**
Working on HL Add Hall Card

### CODE REVIEW

File: src/views/HousingLottery/index.jsx
Comment (andrew-wzj) [inline src/views/HousingLottery/index.jsx:1]: For this file, the branch needs to git pull and merge to housing lottery before placing into the senior project branch

**Review by andrew-wzj — APPROVED** [review]
Looks good to me

File: src/views/HousingLottery/index.jsx
Comment (ArabellaJi) [reply src/views/HousingLottery/index.jsx:1]: Actually, according to what we discussed with Dr. Tuck, we should merge our own branches into senior project first, and then merge the changes in senior project into housing lottery after peer reviews. Does this make sense?

**Review by ahwnsals1234 — COMMENTED** [review]
LGTM!!!

**Pull request merged**